R/configuration.R

```diff
@@ -69,6 +69,12 @@
 #' to the real data. Before record replacement happens in internal
 #' function `write_interactions()`, while before playback replacement
 #' happens in internal function `YAML$deserialize_path()`
+#' 
+#' ## Errors
+#' 
+#' - `verbose_errors` Do you want more verbose errors or less verbose
+#' errors when cassette recording/usage fails? Default is `FALSE`, that is,
+#' less verbose errors.
```

Comment:
Maybe add something about *suggestions* since those won't be printed either, like

* If `TRUE` error messages will include more details about what went wrong and suggest possible solutions (default `FALSE`)

Reply:
yes, will do